swarm/api/api_test.go

```diff
@@ -129,7 +141,26 @@ func TestApiPut(t *testing.T) {
 		}
 		resp := testGet(t, api, addr.Hex(), "")
 		checkResponse(t, resp, exp)
+		testutil.CheckTag(t, tags, chunk.SPLIT, 2, 2) //1 chunk data, 1 chunk manifest
+	})
+}
+
+// TestApiTagLarge tests that the the number of chunks counted is larger for a larger input
+func xTestApiTagLarge(t *testing.T) {
```

Comment:
this is flaking for some reason. my guess is not enough entropy on the random number generator to create distinct chunks. opting in for removal upon agreement regarding the assumption